src/PSModulePublisher/Private/Get-ProjectVariables.ps1

```diff
@@ -7,8 +7,7 @@ $ErrorActionPreference = 'Stop'
 try {
     # Global constants
     $global:PROJECT = @{}
-    $global:PROJECT['SUPERPROJECT_BASE_DIR'] = if (git rev-parse --show-superproject-working-tree) { Convert-Path -Path (git rev-parse --show-superproject-working-tree) }
-    $global:PROJECT['BASE_DIR'] = if ($global:PROJECT['SUPERPROJECT_BASE_DIR']) { $global:PROJECT['SUPERPROJECT_BASE_DIR'] } else { Convert-Path -Path (git rev-parse --show-toplevel) }
```

Comment:
`git` is assumed to be used since afterall repos such as this one are built around `git`

`git rev-parse --show-toplevel` returns present project's root dir if no git superproject exists.

I don't see a need for this change.

Reply:
See #38 #39 for replacement solution.